### ISSUE TITLE
Cannot scroll/drag pinch-zoomed pages

### ISSUE THREAD

**Issue by fabiolatorre** [fastaddons/ScrollAnywhere#46]
**Describe the bug**
Cannot scroll/drag pinch-zoomed pages

**Steps To Reproduce**
1. Go to any supported webpage
2. Use the pinch-to-zoom gesture with the touchpad to zoom in (not the `command`+`+` zoom)
3. Scrolling on the zoomed page doesn't work along any axis. (If the page was originally scrollable, some limited movement happens by scrolling)

**Expected behavior**
The page should be scrolled/dragged consistently.

**Device info:**
 - operating system: macOS 10.15.7
 - Browser and version: Firefox 97
 - addon version: 8.7

**Additional info**
I'm using the middle button. The problem occurs in both `Emulate Scrollbar` and `Grab and Drag` setups. Momentum and Axis locking settings do not seem to affect this behavior.

**Comment by fastaddons** [comment]
Yeah..., this is gonna be very hard :)
I can't find any API that would give me the value of the "pinched" zoom.
Also I'm unable to make a pinch zoom no matter what I try (since I use desktops with no touchpads).
And lastly, this feels like a really rare case, users with touchpad usually scrolls with gestures since those works great. What you are describing is a combination of a hand gesture followed by a mouse interaction? :)

**Comment by fabiolatorre** [comment]
Not really. I mean, I could, but in my case I have mapped the thumb wheel of my Logitech mouse to the zoom function with Logitech Options. I find it pretty handy to quickly zoom in and out while browsing. Sad that there are isn't an API for that. The thing is that scroll bars do appear when I zoom, and that's the reason why I was surprised when it didn't work as I expected it to in the first place. But yeah, I see that this might not be that common.

**Comment by voidray** [comment]
does this also apply to zooming in using CTRL+Mouse wheel? I use than often for websites where the font is small. When I scroll using the middle mouse button the scroll speed doesn't match the mouse movement if zoom is not 100%.
